drivers/ptx-jni/src/main/cpp/source/ptx_utils.cpp

```diff
@@ -28,14 +28,17 @@
 
 CUresult record_events_create(CUevent* beforeEvent, CUevent* afterEvent) {
     CUresult result = cuEventCreate(beforeEvent, CU_EVENT_DEFAULT);
-    LOG_PTX_AND_VALIDATE("cuEventCreate", result);
+    LOG_PTX_AND_VALIDATE("cuEventCreate (beforeEvent)", result);
+    if (result == CUDA_ERROR_OUT_OF_MEMORY) {
+
+    }
```

Comment:
Remove this code. I believe that if result is not Success, the `LOG_PTX_AND_VALIDATE` will display the error code.